distribution/lib/Standard/Table/0.0.0-dev/src/Match_Columns.enso

```diff
@@ -16,146 +7,3 @@ type Match_Columns
 
        Note: column names are not compared.
     By_Position
-
```

Comment:
No meaningful code changes in this file, just moving the helpers to separate module - `Match_Columns_Helpers`.